crates/whiskers/examples/rng.rs

```diff
@@ -0,0 +1,94 @@
+use rand::Rng;
+use std::ops::Range;
+use whiskers::prelude::*;
+
+#[derive(Sketch)]
+struct RngSketch {
+    width: f64,
+    height: f64,
+}
+
+impl Default for RngSketch {
+    fn default() -> Self {
+        Self {
+            width: 400.0,
+            height: 300.0,
+        }
+    }
+}
+
+impl App for RngSketch {
+    fn update(&mut self, sketch: &mut Sketch, ctx: &mut Context) -> anyhow::Result<()> {
+        sketch.stroke_width(3.0);
+
+        let should_generate_random_color = ctx.rng_boolean();
+
+        println!(
+            "Was a random color generated? {}",
+            if should_generate_random_color {
+                "Yes"
+            } else {
+                "No"
+            }
+        );
```

Comment:
I feel this example could be a bit cleaner without any println and with another use of `rng_bool`, e.g. alternate between two stroke width, or drawing or not another shape.